rocketpy/Rocket.py

```diff
@@ -428,7 +489,167 @@ def evaluateStaticMargin(self):
         self.staticMargin.setDiscrete(
             lower=0, upper=self.motor.burnOutTime, samples=200
         )
-        return None
+        # Return self
+        return self
+
+    def evaluateDryInertias(self):
+        """Calculates and returns the rocket's dry inertias relative to
+        the rocket's center of mass. The inertias are saved and returned
+        in units of kg*m².
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        self.dry_I_11 : float
+            Float value corresponding to rocket inertia tensor 11
+            component, which corresponds to the inertia relative to the
+            e_1 axis, centered at the instantaneous center of mass.
+        self.dry_I_22 : float
+            Float value corresponding to rocket inertia tensor 22
+            component, which corresponds to the inertia relative to the
+            e_2 axis, centered at the instantaneous center of mass.
+        self.dry_I_33 : float
+            Float value corresponding to rocket inertia tensor 33
+            component, which corresponds to the inertia relative to the
+            e_3 axis, centered at the instantaneous center of mass.
+        self.dry_I_12 : float
+            Float value corresponding to rocket inertia tensor 12
+            component, which corresponds to the inertia relative to the
+            e_1 and e_2 axes, centered at the instantaneous center of mass.
+        self.dry_I_13 : float
+            Float value corresponding to rocket inertia tensor 13
+            component, which corresponds to the inertia relative to the
+            e_1 and e_3 axes, centered at the instantaneous center of mass.
+        self.dry_I_23 : float
+            Float value corresponding to rocket inertia tensor 23
+            component, which corresponds to the inertia relative to the
+            e_2 and e_3 axes, centered at the instantaneous center of mass.
+
+        Notes
+        -----
+        The e_1 and e_2 directions are assumed to be the directions
+        perpendicular to the rocket axial direction.
+        The e_3 direction is assumed to be the direction parallel to the axis
+        of symmetry of the rocket.
+        RocketPy follows the definition of the inertia tensor as in [1], which
+        includes the minus sign for all products of inertia.
+
+        References
+        ----------
+        .. [1] https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+        """
+        # Compute axes distances
+        noMCM_to_CDM = self.center_of_mass_without_motor - self.centerOfDryMassPosition
+        motorCDM_to_CDM = (
+            self.motor_center_of_dry_mass_position - self.centerOfDryMassPosition
+        )
+
+        # Compute dry inertias
+        self.dry_I_11 = (
+            self.I_11_without_motor
```

Comment:
This is confusing, but it is working!

We should create specific tests to these calculations in the future.

Reply:
Absolutely!